api/test/service/store.test.js

```diff
@@ -51,15 +51,17 @@ test('store/add returns signed url for uploading', async (t) => {
     with: spaceDid,
     nb: { link, size: data.byteLength },
     proofs: [proof]
-    // @ts-expect-error ʅʕ•ᴥ•ʔʃ
   }).execute(connection)
 
-  t.not(storeAdd.error, true, storeAdd.message)
+  if (storeAdd.error) {
```

Comment:
now we need to do this, which is the most annoying part. Because if error, other properties in store* object won't be there